apps/mobile-wallet/src/persistent-storage/wallet.ts

```diff
@@ -107,8 +271,12 @@ export const getStoredWallet = async (error?: string): Promise<WalletMetadata> =
   return walletMetadata
 }
 
+export const getStoredWalletMetadataWithoutThrowingError = () => getWalletMetadata(false)
```

Comment:
That's better, but why not directly use `getWalletMetadata(false)` at this point?

Reply:
because then the code in the component where it is used gets a bit more complicated.

```ts
// now
export const getStoredWalletMetadataWithoutThrowingError = () => getWalletMetadata(false)

const { data: walletMetadata } = useAsyncData(getStoredWalletMetadataWithoutThrowingError)

// after
const { data: walletMetadata } = useAsyncData(useCallback(() => getWalletMetadata(false), []))
```

Not a strong opinion in this, I'll implement the one you prefer the most